frontend/src/features/recommendation/SearchFilter.tsx

```diff
@@ -25,9 +25,15 @@ function SearchFilter() {
     setExpanded(!expanded);
   };
 
+  const isFilterActive = () => {
+    return filterParams.toString() !== '';
+  };
```

Comment:
Parameters unrelated to the search filter may be present in `filterParams`. (For example `?embed=1` recently introduced in #493)

Somehow, we need to declare the parameters related to search filters, to check if one of them is active. Perhaps when initializing `useListFilter()` to make this hook responsible for ignoring other params ?

Reply:
Okay i see the problem.
What I've done is to fix a dictionary of recommendations filter. Parameters `language, date, uploader` are on the url only if a value is selected.
For criteria, the filter is active only if the selected value differs from 50.